code/modules/admin/verbs/admin.dm

```diff
@@ -176,6 +176,10 @@
 
 	for(var/obj/item/W in M)
 		if(!M.dropItemToGround(W))
+			// I hate that this is necessary, but the code is literally just dropping or deleting everything otherwise
+			// people should be allowed to keep their fucking organs
+			if(istype(W, /obj/item/organ) || istype(W, /obj/item/bodypart))
```

Comment:
item abstract tho?

Reply:
It’s an admin verb to drop or delete everything, abstract flags are also given to things like armblades which an admin would want to be gone